label_sync/labels.yaml

```diff
@@ -8,6 +8,12 @@
 ---
 default:
   labels:
+    - color: e11d21
+      description: Categorizes an issue or PR as requesting an API review. See https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md
```

Comment:
nit: can we use the git.k8s.io links here for consistency?

Reply:
sure, done